content/about/about_people.rst

```diff
@@ -506,6 +506,33 @@ PC and is especially interested in Web development.
 
   <br/>
 
+`Elena Torrey`_
+---------------
+
+.. image:: /images/etorrey.jpg
+    :scale: 50%
+    :align: right
+    :alt: Elena Torrey
+
+.. class:: no-breaks
+
+  **OSL Intern | The one with an accent**
+
+Elena is a web development major at Clackamas Community College. Joined OSL to get
+experience in back-end development, learn programming in Python and work in
```

Comment:
"programming in Python, and collaborate" (add oxford comma b/c I believe in the oxford comma; consolidate verbs).